blog/migrations/0006_post_tags_tag_id_index.py

```
@@ -0,0 +1,16 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    """Speed up Post × Tag filters (e.g. tag detail post lists) by indexing tag_id on the M2M table."""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)